### ISSUE TITLE
Youtube download measurement always generates a youtube-download error

### ISSUE THREAD

**Issue by amorphic** [amorphitec/netmeasure#1]
When performing a `YoutubeDownloadMeasurement` to any valid URL, a `youtube-download` error is generated:

```
In [1]: from netmeasure.measurements.youtube_download.measurements import YoutubeDownloadMeasurement
In [2]: y = YoutubeDownloadMeasurement('abc123', 'https://github.com/yt-dlp/yt-dlp/issues/6247')
In [3]: y.measure()
WARNING: Falling back on generic information extractor.
ERROR: Unsupported URL: https://github.com/yt-dlp/yt-dlp/issues/6247
Out[3]: YoutubeDownloadMeasurementResult(id='abc123', errors=[Error(key='youtube-download', description='Download utility could not download file', traceback='\x1b[0;31mERROR:\x1b[0m Unsupported URL: https://github.com/yt-dlp/yt-dlp/issues/6247')], download_rate=None, download_rate_unit=None, download_size=None, download_size_unit=None, url='https://github.com/yt-dlp/yt-dlp/issues/6247', elapsed_time=None, elapsed_time_unit=None)
```

**Comment by amorphic** [comment]
This is due to a known issue with `youtube-dl`: https://github.com/yt-dlp/yt-dlp/issues/6247

It is apparently fixed in [`2023.03.04`](https://github.com/yt-dlp/yt-dlp/releases/tag/2023.03.04)

**Comment by amorphic** [comment]
Fixed in https://github.com/amorphitec/netmeasure/commit/c28b6384ec3af8ffae8ad42e0e8d3e04b5cff7c4
